src/index.js

```diff
@@ -0,0 +1,81 @@
+import Updates from '../modules/updates';
+import AddRemove from '../modules/add-remove';
+import Lists from '../modules/lists';
+import './style.css';
+
+const addRemove = new AddRemove();
+
+const order = (a, b) => a.index - b.index;
+
+const logList = () => {
+  const dash = document.getElementById('dashboard');
+  dash.innerHTML = '';
+  addRemove.items.sort(order).forEach((item) => {
+    const lists = document.createElement('li');
+    lists.className =
+      'list-group-item d-flex justify-content-between align-items-center';
+
+    const child = document.createElement('input');
+    child.type = 'checkbox';
+    child.name = 'name';
+    child.className = 'form-check-input pull-left';
+    child.style.marginRight = '12px';
```

Comment:
Magic numbers: The style.marginRight property is set to a hardcoded value of 12px. Instead, it's better to use a constant or a variable that can be easily changed later if necessary.  What do you think?